askama_shared/src/generator.rs

```diff
@@ -353,9 +353,11 @@ impl<'a> Generator<'a> {
 
     fn write_expr(&mut self, ws: &WS, s: &Expr) {
         self.handle_ws(ws);
-        self.write("writer.write_fmt(format_args!(\"{}\", ");
+        self.write("let askama_expr = &");
```

Comment:
does this need to be a reference?

Reply:
For example, `visit_expr()` might result in `self.x` where `x` is defined as a `String`. We just want to take a reference to that member and pass it to the formatting function.